test/command_line/test_slice_sequence.py

```diff
@@ -58,14 +58,40 @@ def test_slice_sequence_with_first_images_missing(dials_regression, tmpdir):
     )
     assert not result.returncode and not result.stderr
 
-    sliced_exp = ExperimentListFactory.from_json_file(
+    sliced_exp = load.experiment_list(
         tmpdir.join("experiments_5_20_10_20.expt").strpath, check_format=False
     )[0]
     assert sliced_exp.scan.get_image_range() == (10, 20)
     assert sliced_exp.scan.get_array_range() == (9, 20)
     assert sliced_exp.scan.get_oscillation()[0] == pytest.approx(83.35)
 
 
+def test_slice_sequence_to_degree_blocks(dials_data, tmpdir):
+    """Slice data into 10 degree blocks i.e. 17 datasets"""
+    expt = dials_data("l_cysteine_4_sweeps_scaled") / "scaled_30.expt"
+    refl = dials_data("l_cysteine_4_sweeps_scaled") / "scaled_30.refl"
+    procrunner.run(
+        [
+            "dials.slice_sequence",
+            "block_size=10",
+            "output.experiments=sliced.expt",
+            "output.reflections=sliced.refl",
+            expt,
+            refl,
+        ],
+        working_directory=tmpdir,
+    )
```

Comment:
Could you call the command line script `run()` method directly and avoid the `procrunner` indirection?